docs/lesson-plans/week-7.md

```diff
@@ -0,0 +1,692 @@
+similar to the previous statements, so we'll just show you an example of how it works.
+
+Example of update statements:
+```sql
+-- Update columns on the given table name
+UPDATE table_name
+-- Set the column 'column_name' to be some calculation or data. The "=" sign can be used as both assignment and 
+-- comparison.
+    SET column_name = calculation_or_data;
+
+UPDATE table_name
+-- Set columns 1 and 2 to the numbers 1 and 2.
+    SET column1_name = 1, 
+        column2_name = 2,
+-- Set column 3 to the sum of 1 and 2 (yes, you can use the values of columns on the current table to set values -- it 
+-- uses the value before updating column1_name and column2_name though).
+        column3_name = column1_name + column2_name;
+
+UPDATE table_name 
+    SET value = "Hello Jumpstarters!"
+-- A query like the ones above but it only activates on a row with a certain condition.
+    WHERE condition;
+```
+
+An example using the `people` table:
+```sql
+-- Set all signup_date values to now.
+UPDATE people
+    SET signup_date = CURRENT_TIMESTAMP;
+
+-- Set all ages to the user's id.
+UPDATE people
+    SET age = id;
+
+-- Set all bio values to be the person's name and set the names to be "Jumpstarter".
+UPDATE people
+    SET bio = name,
+        name = "Jumpstarter";
+```
+
+#### The DELETE Statement
+
+The `DELETE` statement is the statement used for deleting existing data (CRU**D**). Again, the structure is similar
+to the previous statements. 
+
+>Note: if you want to delete everything from the table without having to drop and recreate 
+>the table, you can use the `TRUNCATE table_name;` statement instead. 
+
+Example of delete statements:
+```sql
+-- Delete statement.
+DELETE FROM table_name
+    WHERE condition;
+
+-- Truncate statement. This deletes ALL data in a table.
+TRUNCATE table_name;
+```
+
+An example using the `people` table:
+```sql
+DELETE FROM people
+    WHERE name = "your friend";
+
+TRUNCATE people;
+```
+
+## Backend + DB Connection
+
+Now to the fun part. We'll be connecting your backend application to the database so that you can stop storing 
+information just in memory!
+
+### JDBC and Working with the DB in Java
+
+Again, the ODBC is software that allows an application to connect with a database. The JDBC is a Java version of that 
+written in Java. To actually start making queries in Java, you'll need to tell the JDBC how to connect with your 
+database.
+
+#### Properties and Environment Files
+
+Most of the time, when working locally, you and other coworkers/teammates/classmates will have different 
+usernames/passwords/urls/other properties which are extremely unlikely to be the same. You also wouldn't want these 
+values to be seen publicly on a Github repository or anything, because then that means anyone with access to the 
+repository will have access to your sensitive information. To deal with this, what most developers do is they create 
```

Comment:
I think this would sound cleaner "To deal with this, developers create property..."